infra/gcp/lib.sh

```diff
@@ -52,6 +52,9 @@ trap 'cleanup_tmpdir' EXIT
 # Useful organization-wide constants
 #
 
+# Set up logs bucket
+export K8S_INFRA_GCSLOGS_BUCKET=gs://k8s-infra-artifacts-gcslogs
```

Comment:
Style nit: everything else in here is quoted so it stands out in syntax highlighting
```suggestion
export K8S_INFRA_GCSLOGS_BUCKET="gs://k8s-infra-artifacts-gcslogs"
```